src/olympia/lib/settings_base.py

```diff
@@ -1798,6 +1798,10 @@ def read_only_mode(env, retry_after=None):
     )
 }
 
+# Change this to deactivate API throttling for views using a throttling class
+# depending on the one defined in olympia.api.throttling.
+API_THROTTLING = True
```

Comment:
There is another `API_THROTTLE` setting in `dev/settings.py` but we're now using `API_THROTTLING`